README.md

```diff
@@ -20,6 +20,7 @@ addon | version | summary
 [base_suspend_security](base_suspend_security/) | 10.0.1.0.0 | Suspend security checks for a call
 [base_technical_features](base_technical_features/) | 10.0.1.0.0 | Access to technical features without activating debug mode
 [base_user_gravatar](base_user_gravatar/) | 10.0.1.0.0 | Synchronize Gravatar Image
+[database_cleanup](database_cleanup/) | 10.0.1.0.0 | Database cleanup
```

Comment:
@eantones You must to change this section into the README since it's automatically generated by a bot. If you undo your changes here, the conflicts will be resolved